go/oasis-test-runner/oasis/client.go

```diff
@@ -61,6 +61,8 @@ func (client *Client) ModifyConfig() error {
 	if len(client.runtimes) > 0 {
 		client.Config.Mode = config.ModeClient
 		client.Config.Runtime.Provisioner = client.runtimeProvisioner
+		client.Config.Runtime.SGXLoader = client.net.cfg.RuntimeSGXLoaderBinary
```

Comment:
Is it possible that `net` or `cfg` is `nil`?

Reply:
Should never be.